readme/Readme.scalatex

```diff
@@ -0,0 +1,324 @@
+
+  @sect{Getting started}
+    @p
+      To include Chimney to your SBT project, add following line to you @code{build.sbt}
+    @hl.scala
+      libraryDependencies += "io.scalaland" %% "chimney" % "0.1.8"
+    @p
+      Library is released for Scala 2.11 and 2.12.
+      If you want to use it with Scala.js, you need to replace @code{%%} with @code{%%%}.
+      Due to some @lnk("compiler bugs", "https://issues.scala-lang.org/browse/SI-7046"),
+      it's recommended to use at least Scala 2.11.9 or 2.12.1.
+
+  @sect{Usage}
+    @p
+      In this section you will learn how to use Chimney example by example.
+
+    @sect{Basic transformations}
+      @p
+        When target object contains only fields present in source object, with corresponding
+        types, we can use shorthanded @code{transformInto}.
+
+      @hl.scala
+        case class Catterpillar(size: Int, name: String)
+        case class Butterfly(size: Int, name: String)
+
+        val stevie = Catterpillar(5, "Steve")
+        val steve = stevie.transformInto[Butterfly]
+        // Butterfly(5, "Steve")
+
+    @sect{Nested transformations}
+      @p
+        It also works when transformation needs to be recursive, possibly involving traversal
+        on nested collection.
+
+      @hl.scala
+        case class Youngs(insects: List[Catterpillar])
+        case class Adults(insects: List[Butterfly])
+
+        val kindergarden = Youngs(List(Catterpillar(5, "Steve"), Catterpillar(4, "Joe")))
+        val highschool = kindergarden.transformInto[Adults]
+        // Adults(List(Butterfly(5, "Steve"), Butterfly(4, "Joe"))
+
+      @p
+        We can use it as long as Chimney can recursively construct transformation for all fields
+        of a target object. In this example transformer for @hl.scala{List} type is constructed
+        basing on automatically derived @hl.scala{Catterpillar ~> Butterfly} mapping.
+
+    @sect{Providing missing values}
+      @p
+        Let's add a field to our @hl.scala{Butterfly} case class.
+
+      @hl.scala
+        case class Butterfly(size: Int, name: String, wingsColor: String)
+
+      @p
+        Now, when trying to perform the same transformation, we get compile-time error. This is
+        naturally expected, as we don't have any data source for new @code{wingsColor} field.
+
+      @hl.scala
+        val stevie = Catterpillar(5, "Steve")
+        val steve = stevie.transformInto[Butterfly]
+        // error: could not find implicit value for parameter derivedTransformer: io.scalaland.chimney.DerivedTransformer[Catterpillar,Butterfly,shapeless.HNil]
+        //        val steve = stevie.transformInto[Butterfly]
+        //                                        ^
+
+      @p
+        In this scenario, we can use Chimney syntax to provide a missing value.
+
+      @hl.scala
+        val steve = stevie.into[Butterfly]
+          .withFieldConst(_.wingsColor, "white")
+          .transform
+        // Butterfly(5, "Steve", "white")
+
```

Comment:
I would add:

Notice, that `.transformInto[T]` is a shortcut for `.into[T].transform`, where the latter form allow us to provide additional transformation rules.